### DISCUSSION TITLE
Update New Issue instructions/boilerplate to remind people to update all packages before posting a new issue

### DISCUSSION THREAD

**Discussion by jeffchown** [livewire/flux#520] [Feature Requests] [answered]
@calebporzio , @joshhanley Thank you for all of your, and your team's, great work on Livewire and Flux!

More and more issues are being posted that are eventually being resolved by the original poster simply updating their Livewire and Flux to the latest version(s).

As you and your team are very responsive and pushing many regular fixes and updates (Flux is getting better every day), it can be easy for people's Livewire and Flux packages to become outdated.

Could you please add a paragraph to the top of the New Issue instructions/boilerplate to ask people to first make sure they have updated Livewire and Flux before posting a new issue? - with some basic recommendations re: best practices, maybe something along these lines:

> Before you post a new issue, please first try the following steps:
> - run `composer update`
> - delete your `resources/views/flux` directory (to make sure you don't have any old/outdated published versions of any Flux components)
> - run `php artisan view:clear`
> - run `npm run build` or `npm run dev`

Thanks!

**Comment by joshhanley** [comment, marked as answer]
Thanks for all your help @jeffchown! I just saw Caleb recently updated the issue template to the below. Hopefully that covers most things, but if there is anything else you think we should add, please let us know 🙂

---

Hi there! 👋

Thank you for using Flux!

Many of the people who help resolve issues here are enthusiastic Fluxers like yourself who volunteer their time to support fellow community members.

To respect everyone's valuable time and help you resolve your issue as quickly as possible, before posting an issue, please:

* Make sure you have followed each step in the installation instructions: https://fluxui.dev/docs/installation (step 5 is optional)
* Update to the latest version of Flux by running `composer update livewire/flux livewire/flux-pro`
* Delete any previously published Flux components (to avoid usage of old versions)
* Run `npm run build` to compile all CSS and JS assets
* Clear the view cache by running `php artisan view:clear`

If these steps do not resolve the issue, please provide us with:

* A concise description of the problem
* Minimal, copy-pastable code (with no dependencies) to reproduce the bug (Volt components are great for this!)
* Screenshots if relevant

We will close this issue for any of the following reasons:

* The code snippets contain unrelated code such as left-over Blade variables
* We cannot reproduce the issue with the provided code snippets
* The issue is a duplicate

Thank you!

**Reply by jeffchown** [reply]
You're welcome, @joshhanley .  Yes, @calebporzio updated the issue template.  All's good.  Will do 👍
